apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -194,6 +194,7 @@ public Patient read(@IdParam IdType patientId, RequestDetails requestDetails) {
   }
 
   @Search
+  @Trace
```

Comment:
This is really the most important trace point here, as pulling the whole call stack out will allow us to see what's being spent on serialization and other HAPI/Spring framework stuff.